Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang
+
+ENV SRCPATH /go/src/github.com/containers/image
+
+# needed because if the tests run as root the tests that test permissions will fail.
+RUN useradd -d /home/gouser -m -s /bin/bash gouser
+RUN apt-get update -qq && apt-get install -y btrfs-tools libdevmapper-dev libgpgme11-dev
+
+COPY . ${SRCPATH}
+WORKDIR ${SRCPATH}
+RUN make deps
+RUN chown -R gouser:gouser ${SRCPATH}
+
+USER gouser
+CMD make .gitvalidation validate test
```

Comment:
the only reason I've been dropping test-skopeo in these changes is because the build is busted atm.

per discussion on #214 these will be reincorporated after the skopeo changes are merged, and before #214 itself is merged.

I will adjust the make target.